demos/bubblewrap/actors/utils.py

```diff
@@ -1,12 +1,22 @@
 import os
```

Comment:
Can we perhaps change the name of the file to something more descriptive? `download_data.py` or similar?